cves/kernel/CVE-2021-3656.yml

```diff
@@ -243,8 +253,10 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
+  - commit: bb00bd9c0862558c6528e3ac97470aee222436ef
+    note: | 
```

Comment:
Make sure these are indented two more spaces - I think this is your yml syntax error

Reply:
(similar issues below too)